### PR TITLE
Improve example usage of Speech to Text

### PR DESCRIPTION
**Pull request by glennrfisher**
This pull request modifies the microphone examples. This should make it easier for users to reference the basic usage with `SpeechToText` or advanced usage with `SpeechToTextSession`. It also adds an example to configure the audio session.

- [x] Add `MicrophoneAdvancedViewController` to demonstrate `SpeechToTextSession`
- [x] Add `MicrophoneBasicViewController` to demonstrate `SpeechToText`
- [x] Add example to configure audio session

### CODE REVIEW

File: Speech to Text/MicrophoneAdvancedViewController.swift [hunk 1]
```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright IBM Corporation 2016
```
Comment (mkistler) [inline Speech to Text/MicrophoneAdvancedViewController.swift]: This should probably be 2018.

File: Speech to Text/MicrophoneBasicViewController.swift [hunk 2]
```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright IBM Corporation 2016
```
Comment (mkistler) [inline Speech to Text/MicrophoneBasicViewController.swift]: I guess this is mostly a rename of `MicrophoneViewController.swift`, so here we might want
> Copyright IBM Corporation 2016, 2018

**Review by mkistler — APPROVED** [review]
Need to fix up the copyright dates.  Otherwise looks good. 👍

**Comment by glennrfisher** [comment]
Great catch, @mkistler! I updated the copyright dates.

Also made some other changes that come to mind:
- [x] Remove NSAppTransportSecurity exception, which is no longer needed.
- [x] Add example for requesting microphone permission
- [x] Add comments to help explain the advanced configuration

**Pull request merged**